src/main/scala/com/zendesk/sbtpillar/Consul.scala

```diff
@@ -0,0 +1,56 @@
+package com.zendesk.sbtpillar
+
+import com.ecwid.consul.v1.QueryParams
+import com.ecwid.consul.v1.health.model.HealthService
+import com.ecwid.consul.v1.health.{HealthClient, HealthConsulClient}
+import sbt.Logger
+
+import scala.collection.JavaConversions._
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import scala.util.{Failure, Success, Try}
+
+class Consul(
+  config: ConsulConfig,
+  isDevOrTestEnv: Boolean,
+  logger: Logger
+) {
+
+  /**
+    * @param serviceName - e.g. "kafka"
+    * @param tag - the tag to filter entries by, generally the pod-id, e.g. "pod100"
+    * @param fallBackValues - values to return if no nodes found in consul, e.g. "localhost:8888"
+    * @return - returns the node:port mappings found in Consul for the service and tag, otherwise returns the fallBackValues
+    */
+  def hostsByServiceName(serviceName: String, tag: Option[String], fallBackValues: List[String]): Future[List[String]] = Future {
+    logger.info(s"Collecting $serviceName nodes from Consul(${config.host}:${config.port}) for pod/tag=${tag.getOrElse("")}")
+    if (isDevOrTestEnv) {
```

Comment:
Probably don't need this fallback as it should fail fast to connect to the local ZDI consul if it isn't running. And if it is running it won't have the hosts listed anyway